Workflow.podspec

```diff
@@ -0,0 +1,23 @@
+Pod::Spec.new do |s|
+  s.name         = 'Workflow'
+  s.version      = '0.1.0'
+  s.summary      = 'Reactive application architecture'
+  s.homepage     = 'https://www.github.com/square/workflow'
+  s.license      = 'Apache License, Version 2.0'
+  s.author       = 'Square'
+  s.source       = { :git => 'https://github.com/square/workflow.git' }
+
+  s.swift_versions = ['4.2']
```

Comment:
if you do use this DSL you probably want to add `s.cocoapods_version >= '1.7.0.beta.1'` to ensure consumers can receive a msg.

Reply:
Alternative you can use `s.swift_version = '4.2'` to begin with.

Reply:
1.7.x will be backwards compatible once you switch to multiple Swift versions